var/target/pr/.README

```diff
@@ -0,0 +1,7 @@
+The /var/target/pr directory is used by the LinuxIO (LIO) framework to
```

Comment:
Did you consider creating this directory as a post-install hook of the package? Or using the delphix-platform service? It seems awkward to use this `.README` file as a mechanism for creating the directory...

Reply:
I've considered it, but I've decided it was better this way since it makes it easier to track where the directory came from using `dpkg -S`.